### PR TITLE
Add support for nerdctl save.

### PR DESCRIPTION
**Pull request by soharaki**
Refs: #33
Signed-off-by: Sho Haraki <endurance.test.20380119@gmail.com>

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
This seems an equivalent of `docker save`, not `docker export`.

`docker export` should produce a bare rootfs tar, while `docker save` should produce Docker Image Spec v1.2 tar.

**Comment by AkihiroSuda** [comment]
Could you change the PR to `nerdctl save` ? Also, I think we are unlikely to support `skip-manifest-json` flag.

File: export.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+package main
+
+import (
+	"io"
+	"os"
+
+	"github.com/containerd/containerd/images/archive"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	terminal "golang.org/x/crypto/ssh/terminal"
```
Comment (AkihiroSuda) [inline export.go:27]: This library is smaller: https://github.com/mattn/go-isatty

File: export.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+func exportContainer(id string, out io.Writer, exportOpts []archive.ExportOpt, clicontext *cli.Context) error {
+	client, ctx, cancel, err := newClient(clicontext)
+	if err != nil {
+		return err
+	}
+	defer cancel()
+
+	container, err := client.LoadContainer(ctx, id)
```
Comment (soharaki) [inline export.go:93]: I'll remove those src.

Comment (soharaki) [inline export.go:93]: I'll remove those lines.

File: export.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	Name:   "export",
+	Usage:  "Export a container's filesystem as a tar archive",
+	Action: exportAction,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    "output",
+			Aliases: []string{"o"},
+			Usage:   "Write to a file, instead of STDOUT",
+		},
+		&cli.BoolFlag{
```
Comment (soharaki) [inline export.go:40]: Remove this option.

**Pull request closed without merging**